app/workers/mood_reminder_worker.rb

```diff
@@ -0,0 +1,12 @@
+class MoodReminderWorker
+  def perform
+    users = User.all
+    users.each do |user|
+      if user.logged_mood?
+        'user logged mood already'
```

Comment:
What purpose does this return value serve?

Reply:
Yeah you could just change this to if !user.logged_mood? and then run the code in the else block